src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/.template.config/localize/templatestrings.cs.json

```diff
@@ -0,0 +1,24 @@
+{
+  "author": "Microsoft",
+  "name": "Blazor Web App",
+  "description": "A project template for creating a Blazor app hosted inside an ASP.NET app that runs on the server. This template can be used for web apps with rich dynamic user interfaces (UIs).",
+  "symbols/Framework/description": "The target framework for the project.",
+  "symbols/Framework/choices/net8.0/description": "Target net8.0",
+  "symbols/skipRestore/description": "If specified, skips the automatic restore of the project on create.",
+  "symbols/ExcludeLaunchSettings/description": "Whether to exclude launchSettings.json from the generated template.",
+  "symbols/kestrelHttpPort/description": "Port number to use for the HTTP endpoint in launchSettings.json.",
+  "symbols/kestrelHttpsPort/description": "Port number to use for the HTTPS endpoint in launchSettings.json. This option is only applicable when the parameter no-https is not used (no-https will be ignored if either IndividualAuth or OrganizationalAuth is used).",
+  "symbols/iisHttpPort/description": "Port number to use for the IIS Express HTTP endpoint in launchSettings.json.",
+  "symbols/iisHttpsPort/description": "Port number to use for the IIS Express HTTPS endpoint in launchSettings.json. This option is only applicable when the parameter no-https is not used (no-https will be ignored if either IndividualAuth or OrganizationalAuth is used).",
+  "symbols/UseClient/displayName": "_Use interactive webassembly components",
+  "symbols/UseClient/description": "If specified, configures the project to render components interactively on the browser using webassembly.",
+  "symbols/UseServer/displayName": "_Use interactive server components",
+  "symbols/UseServer/description": "If specified, configures the project to render components interactively on the server.",
```

Comment:
I wonder if the current descriptions don't make it clear enough that having both WebAssembly and Server interactivity in the same project is a possibility.

Also, what about `--use-wasm` instead of `--use-client`? The term "client" is kind of overloaded. The developer might think "Yeah, of course I want a client for my app!" and use that option without carefully reading the description.

Likewise, `--use-server` might be misinterpreted as "Do you want to create a server project?" I wonder if something like `--use-server-interactivity` would be clearer (although I recognize that's a long name for an option).

Reply:
We need to come up with a meaningful way/name to differentiate between static server rendering and interactive server rendering.

I've changed `--use-client` to `--use-wasm` but I am going to keep `--use-server` until we find a proper name for it. `--use-server-interactivity` is long to type and remember. Specially if we don't have `--use-wasm-interactivity` either.

Reply:
how about getting rid of `use`. like `--wasm` `--server`?

maybe if submiting forms and loading more data doesn't require full page reload one could use `--server-spa` and `--server` to differentiate between interactivity that needs to do full page reload and one that doesn't.

Reply:
I can see dropping the `use` as @itsmuntadhar suggested. It doesn't seem to clarify anything.

I think the description for the Server option is actually pretty decent and clarifying. It says that it "configures the project to render components *interactively* on the server".

Reply:
We can adjust the parameter names later on. I've filed https://github.com/dotnet/aspnetcore/issues/48342 to track the discussion and make a decision.